### PR TITLE
General QA

### PR DESCRIPTION
**Pull request by gsteel**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Gets CI Green Again…

### CODE REVIEW

File: src/EventManager.php [hunk 1]
```diff
@@ -281,7 +282,7 @@ protected function triggerListeners(EventInterface $event, ?callable $callback =
     {
         $name = $event->getName();
 
-        if (empty($name)) {
+        if ($name === null || $name === '') {
```
Comment (samsonasik) [inline src/EventManager.php]: can name be "0" ? see https://getrector.com/demo/66ffa9db-8773-4373-ac55-e91f5a95fcc1

Comment (gsteel) [reply src/EventManager.php]: The doc block is `string|null` for `getName()` - any non-empty-string should be fine, so `"0"` should be ok but I doubt there's a test covering that.

**Review by samsonasik — APPROVED** [review]

File: src/EventManager.php (hunk 1 above)
Comment (gsteel) [reply src/EventManager.php]: On second thoughts, PHP would cast that to `(int) 0` in the array of event names. Still wouldn't be a problem, but better to preserve existing behaviour.

File: doc/book/application-integration/usage-in-a-laminas-mvc-application.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ To register a listener in a laminas-mvc based application, use either applicatio
 
 This example uses module configuration, e.g. `module/Application/config/module.config.php`:
 
-<pre class="language-php" data-line="4-6"><code>
+```php
```
Comment (froschdesign) [inline doc/book/application-integration/usage-in-a-laminas-mvc-application.md:62]: @gsteel 
You have removed the line highlighting: https://docs.laminas.dev/laminas-eventmanager/application-integration/usage-in-a-laminas-mvc-application/#register-listener

**Pull request merged**